homeassistant/components/rointe/sensor.py

```diff
@@ -0,0 +1,108 @@
+"""A sensor for the current Rointe radiator temperature."""
+from __future__ import annotations
+
+from datetime import datetime
+
+from rointesdk.device import RointeDevice
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ENERGY_KILO_WATT_HOUR, POWER_WATT, TEMP_CELSIUS
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from .const import DOMAIN
+from .coordinator import RointeDataUpdateCoordinator, RointeSensorEntityDescription
+from .rointe_entity import RointeRadiatorEntity
+
+SENSOR_DESCRIPTIONS = [
+    # Current room temperature sensor (probe value).
+    RointeSensorEntityDescription(
+        key="current_temperature",
+        name="Current Temperature",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        native_unit_of_measurement=TEMP_CELSIUS,
+        state_class=SensorStateClass.MEASUREMENT,
+        value_fn=lambda radiator: radiator.temp_probe,
+        last_reset_fn=lambda radiator: None,
+    ),
+    # Energy usage in Kw/h.
+    RointeSensorEntityDescription(
+        key="energy",
+        name="Energy Consumption",
+        device_class=SensorDeviceClass.ENERGY,
+        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        state_class=SensorStateClass.TOTAL,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda radiator: radiator.energy_data.kwh
+        if radiator.energy_data
+        else None,
```

Comment:
We shouldn't have inline ternary statements if they break like this. You could always make a separate function for this description's `value_fn` kwarg.

Reply:
I'm almost sure this was done by `black`. Still, I changed it to a function.